src/adapter/src/rbac.rs

```diff
@@ -560,6 +565,10 @@ fn generate_required_ownership(plan: &Plan) -> Vec<ObjectId> {
             .unwrap_or_default(),
         // Do not need ownership of descendant objects.
         Plan::DropObjects(plan) => plan.referenced_ids.clone(),
+        Plan::AlterClusterRename(plan) => vec![ObjectId::Cluster(plan.id)],
+        Plan::AlterClusterReplicaRename(plan) => {
+            vec![ObjectId::ClusterReplica((plan.cluster_id, plan.replica_id))]
```

Comment:
Could add RBAC tests to check if renaming them works/fails as expected. (I'm kind of wondering that this shows up as uncovered, we should have RBAC enabled by default in CI)

Reply:
That's a good point. I think we're missing a discussion about which privileges would be required to rename clusters/replicas. I tried to add tests, but the crux is that I don't know whether renaming should require a specific privilege or not. @jkosh44 let's talk about this once you're back!

Reply:
Renaming an object (and all other alter's) is the equivalent of dropping that object and then re-creating it from a privilege point of view. That means for clusters you need:
- Ownership of the cluster.
- The `CREATECLUSTER` attribute.
For cluster replicas you need:
- Ownership of the replica.
- `CREATE` privileges on the containing cluster.

Reply:
Wait sorry, I think I'm mistaken here. It only requires Ownership for both. Sorry about that.